### PR TITLE
Fill in SDK gaps

### PR DESCRIPTION
**Pull request by pranavkm**
* Adds missing details to RazorSDK
* Adds a Web SDK section (cc @vijayrkn)
* Re-organize TOC a bit

### CODE REVIEW

File: aspnetcore/razor-pages/sdk.md [hunk 1]
```diff
@@ -121,6 +121,10 @@ The properties and items in the following table are used to configure inputs and
 | `EmbedRazorGenerateSources` | When `true`, adds RazorGenerate (*.cshtml*) items as embedded files to the generated Razor assembly. Defaults to `false`. |
 | `UseRazorBuildServer` | When `true`, uses a persistent build server process to offload code generation work. Defaults to the value of `UseSharedCompilation`. |
 | `GenerateMvcApplicationPartsAssemblyAttributes` | When `true`, the SDK generates additional attributes used by MVC at runtime to perform application part discovery. |
+| `DefaultWebContentItemExcludes` | A globbing pattern for item elements that are to be excluded from the `Content` item group in projects targeting the Web or Razor SDK |
+| `ExcludeConfigFilesFromBuildOutput` | When `true`, *.config* and *.json* files do not get copied to the build output directory. |
+| `AddRazorSupportForMvc` | When `true`, configure the Razor SDK to add support for the MVC configuration that is required when building applications containing MVC views or Razor Pages. This property is implicitly set for .NET Core 3.0 or later projects targeting the Web SDK |
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/sdk.md]: | `AddRazorSupportForMvc` | When `true`, configure the Razor SDK to add support for the MVC configuration that is required when building applications containing MVC views or Razor Pages. This property is implicitly set for .NET Core 3.0 or later projects targeting the Web SDK |

**Comment by Rick-Anderson** [comment]
@pranavkm  let me know when I can do an edit pass. Great info but needs minor editing.

**Comment by pranavkm** [comment]
All yours 👍

**Comment by Rick-Anderson** [comment]
Do we need @vijayrkn to sign off or can I S&M?

**Comment by vijayrkn** [comment]
> Do we need @vijayrkn to sign off or can I S&M?

LGTM

**Pull request merged**